src/lib/core/data-table/data-source.ts

```diff
@@ -14,4 +14,5 @@ export interface CollectionViewer {
 
 export abstract class DataSource<T> {
   abstract connect(collectionViewer: CollectionViewer): Observable<T[]>;
+  disconnect() {}
```

Comment:
Makes sense to me, but it does require that all data sources need to implement it now. At least a no-op